docs/Operators.md

```diff
@@ -11241,3 +11248,37 @@ expect(node, inputs=[x], outputs=[y],
 </details>
 
 
+### Functions: 
+*Functions in ONNX is defined as optional composed operators. Frameworks can support it directly or via decomposing it into simple operators.*
+### <sub>experimental</sub> <a name="FuncMeanVarianceNormalization"></a><a name="funcmeanvariancenormalization">**MeanVarianceNormalization**</a>
+
+  A MeanVarianceNormalization Function: Perform mean variance normalization on the input tensor X using formula: <br/> ``` (X-EX)/sqrt(E(X-EX)^2) ``` <br/>
+  #### Inputs
+  
+   <dl>
+  <dt><tt>X</tt>(float/float16/double)</dt>
+  <dd>with shape [N,C,W,H] or N-D shape</dd>
+  </dl>
+  
+  #### Attributes
+  
+   <dl>
+  <dt><tt>axes</tt>(ints)</dt>
+  <dd>will be passed to ReducedMean Ops. Use [0,2,3] (without C axis for N-D cases) for calculating means and variances along channels. Two variables with the same C-coordinate are associated with the same mean and variance. Use [0,1,2,3] (with C axis) to calculate global mean and global variance with all variables sharing the same mean/variance.</dd>
+  <dt><tt></tt></dt>
+  <dd>The KeepDims attribute in ReducedMean is set to true for calculation</dd>
+  </dl>
+  
+  #### Outputs
+  
+   <dl>
+  <dt><tt>X_MVN</tt>(float/float16/double)</dt>
+  <dd>with the same shape as input X</dd>
+  </dl>
+  
+
+#### Version
```

Comment:
please follow the same pattern as operators -- putting Version section as the first one.